application/src/main/java/org/exoplatform/chat/portlet/notification/templates/index.gtmpl

```diff
@@ -10,6 +10,7 @@
   data-chat-interval-notif="<%=chatIntervalNotif%>"
   data-chat-interval-status="<%=chatIntervalStatus%>"
   data-short-space-name="<%=shortSpaceName%>"
+  data-db-name="<%=dbName%>"
```

Comment:
Even when user changes it, he cannot get information from other databases or other users because to read database, he need other informations: username, token.